### PR TITLE
Add lldb to alpine 3.12/3.13 images

### PR DESCRIPTION
**Pull request by mikem8361**

### CODE REVIEW

**Review by hoyosjs — APPROVED** [review]

**Comment by hoyosjs** [comment]
@mthalman could you please take a look?

File: src/alpine/3.12/helix/amd64/Dockerfile [hunk 1]
```diff
@@ -25,6 +25,7 @@ RUN apk update && \
         libffi-dev \
         linux-headers \
         llvm \
+        lldb-dev \
```
Comment (mthalman) [inline src/alpine/3.12/helix/amd64/Dockerfile:28]: Please list this in alphabetical order with the other packages.  Same applies to the Alpine 3.13 Dockerfile.

Comment (mikem8361) [reply src/alpine/3.12/helix/amd64/Dockerfile:27]: Done.

**Comment by mikem8361** [comment]
@mthalman can we merge this and create/push the images? Thanks.

**Comment by mikem8361** [comment]
@mthalman when can this PR be merged and the images published?  Our diagnostics testing is blocked on taking the latest 6.0 runtime updates on the newer Alpine version with lldb.

**Review by mthalman — APPROVED** [review]

**Pull request merged**